packages/docs/stories/components/button.stories.ts

```diff
@@ -1,268 +1,226 @@
-<syn-button size="medium" pill>Medium</syn-button>
-<syn-button size="large" pill>Large</syn-button>`,
-};
-
-/**
- * Use the circle attribute to create circular icon buttons. When this attribute is set, the button expects a single <syn-icon> in the default slot.
- */
-export const CircleButtons: Story = {
-  render: () => html`<syn-button variant="default" size="small" circle>
-  <syn-icon name="gear" label="Settings"></syn-icon>
-</syn-button>
-
-<syn-button variant="default" size="medium" circle>
-  <syn-icon name="gear" label="Settings"></syn-icon>
-</syn-button>
-
-<syn-button variant="default" size="large" circle>
-  <syn-icon name="gear" label="Settings"></syn-icon>
-</syn-button>`,
+  parameters: {
+    docs: {
+      description: generateStoryDescription('size'),
+    },
+  },
+  render: () => html`
+  <syn-button size="small">Small</syn-button>
+  <syn-button size="medium">Medium</syn-button>
+  <syn-button size="large">Large</syn-button>
+  <style>
+    syn-button {
+      margin: 0.2rem;
+    }
+  </style>`,
 };
 
-/**
- * Use the text variant to create text buttons that share the same size as regular buttons but don't have backgrounds or borders.
- */
-export const TextButtons: Story = {
-  render: () => html`<syn-button variant="text" size="small">Text</syn-button>
-<syn-button variant="text" size="medium">Text</syn-button>
-<syn-button variant="text" size="large">Text</syn-button>`,
+/** The focus attribute provides feedback to the users,
+ * informing them that the button component is ready for use.  */
+export const Focus: Story = {
+  play: ({ canvasElement }: { canvasElement: HTMLElement }) => {
+    const button = canvasElement.querySelector('syn-button') as HTMLInputElement;
+    if (button) {
+      button.focus();
+    }
+  },
+  render: () => html`<syn-button>Default</syn-button>`,
 };
 
-/**
- * It's often helpful to have a button that works like a link. This is possible by setting the href attribute, which will make the component render an <a> under the hood. This gives you all the default link behavior the browser provides (e.g. [[CMD/CTRL/SHIFT]] + [[CLICK]]) and exposes the target and download attributes.
- */
 export const LinkButtons: Story = {
-  render: () => html`<syn-button href="https://example.com/">Link</syn-button>
-<syn-button href="https://example.com/" target="_blank">New Window</syn-button>
-<syn-button href="/assets/images/wordmark.svg" download="synergy.svg">Download</syn-button>
-<syn-button href="https://example.com/" disabled>Disabled</syn-button>`,
+  parameters: {
+    docs: {
+      description: generateStoryDescription('link'),
+    },
+  },
+  render: () => html`
+  <syn-button href="https://example.com/">Link</syn-button>
+  <syn-button href="https://example.com/" target="_blank">New Window</syn-button>
+  <syn-button href="/assets/images/wordmark.svg" download="synergy.svg">Download</syn-button>
+  <syn-button href="https://example.com/" disabled>Disabled</syn-button>
```

Comment:
Error: Disabled links with a href should not only be displayed as disabled, but also not change the location. Currently, when clicking on the Disabled Link, it will still open example.com.

Reply:
`<a>` doesn't have a disabled attribute in HTML. Disabled links usually just don't get an `href` to be correctly (non-)accessible. We should just remove this example here completely. If people don't want that a link is clickable, they should remove the link as well, otherwise we would have to build a lot more on top just for the link, e. g. making it not focusable etc.

Reply:
That is the reason why I think it may be better to have two different components for that, which could also be done by sharing a render template. But we already talked about this once I think. Point is, people will eventually run into this and wonder why it does not work.

A very simple fix for this would be something like:

```css
syn-button[href][disabled] {
  pointer-events: none;
}
```

Reply:
Then it would be still accessible via keyboard which is not correct from a11y perspective.

Reply:
It might make more sense to provide a PR for that to only set `href` if the button is not disabled. What do you think about that?

Reply:
Thats also a good point. However, there seem to different arguments about this too:

> If disabled buttons are not focusable with TAB, our user would occasionally miss that the button ever exists, and they will never find it.

We should at least be thinking about this. Like I said, we will get this as an error eventually more sooner than later.

Reply:
> more

Sorry, we seem to have been typing at the same time :). Sure, we can do that. Mind to create the ticket for it?

Reply:
> If disabled buttons are not focusable with TAB, our user would occasionally miss that the button ever exists, and they will never find it.

This is how every button in HTML works. If a button is disabled it's not focusable. If a link doesn't have a href, it's not focusable as well. The problem is here more setting a href and making it disabled at the same time. (To be honest, I'm not sure, if this will ever be a use case. :D)

Reply:
Right. But as you have pointed out, this is not a button, but a link. Which is kind of the problem :)

Usecase:

Wizard Form (Step 1, Step 2, Step 3)
You are on Step 2 (part of a bigger Form) and Form validation does not permit you to move to /account/create/step3

So seen in various open source ecommerce platforms like Magento, or parts of Shopify. Both platforms would try to carry the Session-Data to the next page to prevent issues when reloading.

Reply:
Sooooo.... should I change something still? 😆

Reply:
No, as @mariohamann has pointed out, we will tackle this issue in a new ticket.

Reply:
thank you for clarifying :)

Reply:
@Gwenevere  But you should remove the example from the docs, as they are misleading.

@schilchSICKAG What do you think about improving our docs on dev side for the disabled state to make that more clear? (`If you need disabled links, set `disabled` and do not set the `href`.) But yeah, let's tackle this in an iteration and let's write the ticket open enough, because there are several solutions (e. g. docs, PR, adding pointer-events to prevent at least mouse clicks etc.). :)

Reply:
Seems like a good idea. We can define the ticket tomorrow together, makes it easier :)